js/runner.go

```diff
@@ -131,6 +131,10 @@ func NewFromBundle(
 
 	err = r.SetOptions(r.Bundle.Options)
 
+	if r.Bundle.Options.TLSAuth != nil && r.Bundle.Options.TLSAuth.NameToCertificate != nil {
+		logger.Warn("Since NameToCertificate had been deprecated in golang, it is recommended to leave TLSAuth.Domains empty.")
```

Comment:
I meant to use the sync.Once here, and yes, it should be fine to have a dedicated sync.Once.
```suggestion
nameToCertWarning.Do(func() {
		logger.Warn("tlsAuth.domains option could be removed in the next releases, it's recommended to leave it empty and let k6 automatically detect from the provided certificate. It follows the Go's NameToCertificate deprecation - https://pkg.go.dev/crypto/tls@go1.17#Config.")
		})
```